src/docfx/config/ops/OpsAccessor.cs

```diff
@@ -77,13 +77,21 @@ public async Task<string> GetMarkdownValidationRules((string repositoryUrl, stri
 
         public async Task<string> GetAllowlists((string repositoryUrl, string branch) tuple)
         {
-            return await FetchValidationRules($"/route/validationmgt/validation/allowlists", tuple.repositoryUrl, tuple.branch);
+            return await FetchValidationRules(
+                   $"/taxonomies/simplified?" +
+                   $"name=ms.author&name=ms.devlang&name=ms.prod&name=ms.service&name=ms.topic&name=devlang&name=product",
+                   tuple.repositoryUrl,
+                   tuple.branch);
         }
 
         public async Task<string> GetMetadataSchema((string repositoryUrl, string branch) tuple)
         {
             var metadataRules = FetchValidationRules($"/route/validationmgt/rulesets/metadatarules", tuple.repositoryUrl, tuple.branch);
-            var allowlists = FetchValidationRules($"/route/validationmgt/validation/allowlists", tuple.repositoryUrl, tuple.branch);
+            var allowlists = FetchValidationRules(
```

Comment:
that doesn't work either, it's actually visiting https://op-build-prod.azurewebsites.net/taxonomies/simplified?name=ms.author&name=ms.devlang&name=ms.prod&name=ms.service&name=ms.topic&name=devlang&name=product

Reply:
OK.